src/lib.rs

```diff
@@ -333,6 +333,18 @@ For more info: https://github.com/LiquidityC/Blightmud/issues/173"#;
                         .unwrap();
                 }
             }
+            Event::SetPromptCursorPos(pos) => {
+                if let Ok(mut buffer) = session.command_buffer.lock() {
+                    buffer.set_pos(pos);
+                    session
+                        .main_writer
+                        .send(Event::UserInputBuffer(
+                            buffer.get_buffer(),
+                            buffer.get_pos(),
+                        ))
```

Comment:
I added `Event::UserInputCursor` in anticipation of this API as part of https://github.com/Blightmud/Blightmud/pull/738. I think it'd be better to use that here since the buffer contents haven't changed, we just want a redraw of the prompt at the new cursor pos.

```suggestion
                        .send(Event::UserInputCursor(buffer.get_pos()))
```

Reply:
Done